_includes/dotnet/objects.md

```diff
@@ -95,12 +95,13 @@ string playerName = gameScore.Get<string>("playerName");
 bool cheatMode = gameScore.Get<bool>("cheatMode");
 ```
 
-The three special values are provided as properties:
+The four special values are provided as properties:
 
 ```cs
 string objectId = gameScore.ObjectId;
 DateTime? updatedAt = gameScore.UpdatedAt;
 DateTime? createdAt = gameScore.CreatedAt;
+ParseACL? parseACL = gamescore.ParseACL;
```

Comment:
Typo. Should be **gameScore**. According to API reference and other docs, the property is **ACL**, not **ParseACL**.

See here: http://parseplatform.org/Parse-SDK-dotNET/api/